src/components/site-form.tsx

```diff
@@ -352,6 +395,50 @@ export const SiteForm = ( {
 											value={ sitePath }
 											onClick={ onSelectPath }
 										/>
+										{ wpVersionsEnabled && allowVersionsChange && (
+											<div className="grid grid-cols-2 gap-4 mt-4">
+												<div className="flex flex-col gap-1.5 leading-4">
+													<label className="font-semibold">{ __( 'PHP version' ) }</label>
+													<SelectControl
+														value={ phpVersion }
+														options={ SupportedPHPVersionsList.map( ( version ) => {
+															return {
+																label: version,
+																value: version,
+															};
+														} ) }
+														onChange={ ( value ) => {
+															if ( setPhpVersion ) {
+																setPhpVersion( value as SupportedPHPVersion );
+															}
+														} }
+														__next40pxDefaultSize
+													/>
+												</div>
+												<div className="flex flex-col gap-1.5 leading-4">
+													<label className="font-semibold">{ __( 'WordPress version' ) }</label>
+													<SelectControl
+														value={ wpVersion }
+														options={
+															wpVersions.length > 0
+																? wpVersions.map( ( v ) => ( { label: v.name, value: v.version } ) )
```

Comment:
When you merge trunk you'll be able to pass the wpVersions object without any transformation.

Reply:
Updated! Thanks for catching this one